index.js

```diff
@@ -45,37 +106,10 @@ describe('TEST SUITE', () => {
 
   //require('./tests/history.js'); // Only run against history node.
 
-  require('./tests/testnet-smoketest.js'); // Testnet smoketest. By default runs against local build.
-
   /**
-   * Locked token tests (FIP-6,21). Tests may require additional configuration.
+   * Testnet smoketest. By default runs against local build.
    */
-  //require('./tests/locks-transfer-locked-tokens-max-load.js');  // OPTIONAL PERFORMANCE TEST. Loads the chain with lots of general locks. Run this before other general locks tests when its desirable to test a loaded chain.
-  //require('./tests/locks-transfer-locked-tokens-large-grants.js'); //FIP-21 tests for FIO genesis locks functionality. Allocates large grants from faucet.
-  //### These Lock tests do NOT require additional configuration.
-  require('./tests/locks-transfer-locked-tokens-account-tests.js');  // FIP-6 tests of generic account functionality
-  require('./tests/locks-transfer-locked-tokens.js');  //FIP-21 locking tests for general locks
-  //### These Lock tests require additional configuration.
-  //require('./tests/locks-mainnet-locked-tokens-lock1hotfix.js'); //Release 2.4.1 Hotfix for Type 1 locks (was not calculating voting power correctly)
-  //require('./tests/locks-mainnet-locked-tokens.js'); //FIP-21 tests for FIO genesis locks functionality.
-  //### Testnet only. Not sure this will work. May need updates for recent lock changes
-  //require('./tests/locks-transfer-locked-tokens-testnet-smoke-tests.js'); //Only works for Testnet
-
-  /**
-   * Staking Tests (FIP-21)
-   * May require additional configuration (see the notes in js files before running these tests)
-   * in addition to this we need to possibly develop more tests for checking voting power when accounts have staked
-   */
-  //###These Staking tests do NOT require additional configuration.
-  require('./tests/stake-general-locked-tokens.js'); //FIP-21 tests for general lock accounts performing staking
-  require('./tests/stake-BD-3552-dev-tests.js');
```

Comment:
It looks like some of these tests are getting deleted here but not being added back anywhere? index-staking.js is being shown as deleted above.